libmatekbd/matekbd-indicator.c

```diff
@@ -414,10 +410,7 @@ matekbd_indicator_prepare_drawing (MatekbdIndicator * gki, int group)
 			ln2cnt_map = NULL;
 		}
 
-		gtk_container_add (GTK_CONTAINER (align), label);
-		gtk_container_add (GTK_CONTAINER (ebox), align);
-
-		gtk_container_set_border_width (GTK_CONTAINER (align), 2);
```

Comment:
I guess setting the border on the event box which contains the label would have been the same, but margin is likely just fine too yeah.